### ISSUE TITLE
تشخیص نوع گل بر اساس ویژگی‌های فیزیکی

### ISSUE THREAD

**Issue by hamedhero** [hamedhero/simple-blockchain#1]
from sklearn.datasets import load_iris
from sklearn.model_selection import train_test_split
from sklearn.ensemble import RandomForestClassifier
from sklearn.metrics import accuracy_score

# بارگذاری دیتاست معروف آیریس
data = load_iris()
X = data.data  # ویژگی‌ها (features)
y = data.target  # برچسب‌ها (labels)

# تقسیم داده‌ها به آموزش و تست (۷۰٪ آموزش، ۳۰٪ تست)
X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.3, random_state=42)

# تعریف مدل یادگیری ماشین (جنگل تصادفی)
model = RandomForestClassifier()

# آموزش مدل روی داده‌های آموزش
model.fit(X_train, y_train)

# پیش‌بینی روی داده‌های تست
y_pred = model.predict(X_test)

# محاسبه دقت مدل
accuracy = accuracy_score(y_test, y_pred)
print("Accuracy:", accuracy)

**Comment by hamedhero** [comment]
از sklearn.datasets.load_iris() بارگذاری می‌شود.

این دیتاست شامل ۱۵۰ نمونه گل است از ۳ گونه مختلف:

Setosa

Versicolor

Virginica

هر نمونه دارای ۴ ویژگی عددی است:

Sepal length (طول کاسبرگ)

Sepal width (عرض کاسبرگ)

Petal length (طول گلبرگ)

Petal width (عرض گلبرگ)
